ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/perception/RDXStancePoseSelectionPanel.java

```diff
@@ -178,11 +205,17 @@ public void renderImGuiWidgets()
       {
          PerceptionDebugTools.printMat("Height Map", terrainMapData.getHeightMap(), 4);
       }
-      ImGui.text("World Point: " + latestPose.getTranslation().toString("%.3f"));
+      ImGui.text("World Point: " + latestPickPoint.getTranslation().toString("%.3f"));
       if (terrainMapData != null)
       {
-         ImGui.text("Height: " + terrainMapData.getHeightInWorld(latestPose.getTranslation().getX32(), latestPose.getTranslation().getY32()));
-         ImGui.text("Contact Score: " + terrainMapData.getContactScoreInWorld(latestPose.getTranslation().getX32(), latestPose.getTranslation().getY32()));
+         ImGui.text("Height: " + terrainMapData.getHeightInWorld(latestPickPoint.getTranslation().getX32(), latestPickPoint.getTranslation().getY32()));
+         ImGui.text("Contact Score: " + terrainMapData.getContactScoreInWorld(latestPickPoint.getTranslation().getX32(), latestPickPoint.getTranslation().getY32()));
+      }
+
+      ImGui.checkbox(labels.get("Calculate Stance Pose"), calculateStancePose);
```

Comment:
Render parameters to the UI for tuning, once its tuned its fine but nice to have available